tests/datatypes/test_int.py

```diff
@@ -38,34 +38,37 @@ class BinaryIntOperationTests(BinaryOperationTestCase, TranspileTestCase):
 
     not_implemented = [
         'test_modulo_complex',
-
-        'test_multiply_complex',
-
-        'test_power_complex',
-
-        'test_true_divide_complex',
     ]
 
 
 class InplaceIntOperationTests(InplaceOperationTestCase, TranspileTestCase):
     data_type = 'int'
 
-    not_implemented = [
-
-
-        'test_modulo_complex',
-        'test_modulo_float',
-
-        'test_multiply_bytearray',
-        'test_multiply_bytes',
-        'test_multiply_complex',
-        'test_multiply_float',
-        'test_multiply_list',
-        'test_multiply_tuple',
-
-        'test_power_complex',
-
-
-
-        'test_true_divide_complex',
-    ]
+    substitutions = {
```

Comment:
@freakboy3742 I don't see anything related to what you mention. 

https://github.com/pybee/voc/blob/master/tests/utils.py#L263 or https://github.com/pybee/voc/blob/master/tests/utils.py#L296 

It seems to require substitutions. I thought you meant there was some automatic rounding of results to a given precision.